### ISSUE TITLE
Issue with exporting multiple frames as GAMESS $DATA input

### ISSUE THREAD

**Issue by joanimato** [brettbode/wxmacmolplt#109]
### Please describe the problem or enhancement:
When exporting multiple frames (e.g. an LLM) as a GAMESS data group results in incorrect cartesian coordinates for each frame (in a few occasions frames also seem to be duplicated). 

### If this is a bug report please list the steps required to reproduce the problem:
I was just creating an LLM path and tried to export all frame coordinates at once using the export function. 

### What version are you running?
V7.7

### What OS are you using? (please include 32/64 bit and if Linux the specific distribution)
Linux 64 bit: Linux Mint 18

**Comment by brettbode** [comment]
Hello,
   Sorry about the long delay in replying to this report. It has been a long time since I have been able to put much time into improving the program.
   I have identified a bug with the export code that I believe solves this report. It certainly results in a duplicated initial geometry and then as a result a missing final geometry. I believe I have fixed it in a check today so it will be in the next release.
